__tests__/spec/plugin-validator.js

```diff
@@ -3,76 +3,95 @@
 const path = require('path')
 const cliPath = path.join(__dirname, '..', '..', 'bin', 'cli')
 const { exec } = require('child_process')
+const ansiColors = require('ansi-colors')
 
-function runShellCommand (fixtureProjectDirectory) {
+function runShellCommand (fixtureDirectoryName) {
+  const fixtureProjectDirectory = path.join(__dirname, '..', 'fixtures', 'mockPlugins', fixtureDirectoryName)
   return new Promise((resolve, reject) => {
-    exec(`"${process.execPath}" ${cliPath} validate-plugin ${fixtureProjectDirectory}`,
-      { cwd: fixtureProjectDirectory, env: process.env, stdio: 'inherit' }, function (err, stdout, stderr) {
+    const execResult = exec(`"${process.execPath}" ${cliPath} validate-plugin ${fixtureProjectDirectory}`,
+      { env: process.env, stdio: 'inherit' }, function (err, stdout, stderr) {
+        const output = {
+          stdout,
+          stderr,
+          exitCode: execResult.exitCode
+        }
+
         if (err) {
```

Comment:
This is so that `standardjs` doesn't complain that we're not handling the error :(